plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -232,6 +232,7 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       )
     }
     gesturesManager.rotateGestureDetector.angleThreshold = 3.0f
+    gesturesManager.shoveGestureDetector.maxShoveAngle = 45.0f
```

Comment:
could we add a unit test?

Reply:
could we move 45.0f and 3.0f to constants, mark them as internal and make unit test easier?